tests/e2e/specs/in-game/layout.spec.js

```diff
@@ -228,7 +232,7 @@ describe.skip('Aesthetic tests', () => {
     cy.get('[data-player-hand-card]').should('have.length', 2);
   });
 
-  it('Quadruple jacks with a few cards', () => {
+  it.skip('Quadruple jacks with a few cards', () => {
```

Comment:
This is failing due to a grid issue, an element is overlaying the element we need to click.